openadapt/share.py

```diff
@@ -0,0 +1,110 @@
+"""Copy a recording from one computer to another.
+
+Usage:
+    python -m openadapt.share send --recording_id=1 
+    python -m openadapt.share receive --wormhole_code=<wormhole_code>
```

Comment:
Is it possible to simply run:

```
python -m openadapt.share receive <wormhole_code>
```

Reply:
Yup that works :), I updated the comment to reflect that

Reply:
addressed in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/3434a00b4134bd756043123a23458b3e1c4126ed